template.yaml

```diff
@@ -379,6 +379,28 @@ Resources:
       Action: lambda:InvokeFunction
       Principal: 'apigateway.amazonaws.com'
 
+  CreateSerialPublicationFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: no.sikt.nva.pubchannels.handler.create.serialpublication.CreateSerialPublicationHandler::handleRequest
+      Policies:
+        - !GetAtt GetItemDynamoDbPolicy.PolicyArn
+        - !GetAtt ReadSecretsPolicy.PolicyArn
+      Events:
+        CreateJournalEvent:
+          Type: Api
+          Properties:
+            RestApiId: !Ref PublicationChannelsApi
+            Path: /serial-publication
```

Comment:
In openapi scheme you are defining this endpoint with path   `/serial-publication/{identifier}/{year}`

Reply:
Uups, good catch! Fixed